omicron/cli/process.py

```diff
@@ -58,23 +58,27 @@
 *automatically* submitted to condor for processing.
 
 """
-
+import time
+prog_start = time.time()
```

Comment:
Should this line be later in the code or contained in some function? I'm not sure I understand the behaviour when there are other functions in this module.

Reply:
The idea is to report the time it took for the program to run.  Since the import statements take a significant amount of time I want this statement to record start time as soon as possible.

Reply:
Ok, that's fine. I'll resolve this then